packages/trader/src/sass/app/modules/reports.scss

```diff
@@ -79,8 +79,8 @@ $side-padding: 1.2em;
     /* postcss-bem-linter: ignore */
     .statement__row--detail {
         overflow: hidden;
-        max-height: 95px;
-        height: 95;
+        max-height: 63px;
```

Comment:
why do we need max-height here?

Reply:
I just reverted it to the previous value 😄